sources/packages/web/src/views/student/StudentPDBanner.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <Message
+    severity="warn"
+    :closable="false"
+    v-if="pdStatus === StudentPDStatus.Pending.toString()"
```

Comment:
why `toString()`? enum is already in string 
![image](https://user-images.githubusercontent.com/77353155/140804163-d323064a-5c55-4f5a-a1d1-6c8acd5b0ebb.png)

Reply:
Feedback comment addressed.